commit/chainfee/types.go

```diff
@@ -23,6 +23,9 @@ type Query struct {
 type Outcome struct {
 	// Each Gas Price is the combination of Execution and DataAvailability Fees using bitwise operations
 	GasPrices []cciptypes.GasPriceChain `json:"gasPrices"`
+
+	InflightChainFeeUpdates map[cciptypes.ChainSelector]time.Time `json:"inflightChainFeeUpdates"`
+	InflightRemainingChecks int64                                 `json:"inflightRemainingChecks"`
```

Comment:
I don't see any feature flag to prevent using these new fields until all nodes have been updated.

Reply:
Ah actually I think we'll have a problem with these Outcome type updates even if the feat flag is there, if only a subset of nodes are updated, unless protobuf encodes an outcome `Outcome{GasPrice}` identically to an outcome `Outcome{GasPrices, rest not specified}`. 

The approach that would seem to work is to feat flag the usage of these fields via offchainConfig, upgrade all the nodes (eventually - via canary or whatever process) and then switch this flag on.

Reply:
Need to double check that protobuf has an implicit "omitEmpty" to ensure `Outcome{GasPrices, rest not specified}` is encoded the same as `Outcome{GasPrice}`.

I'm going to grab some sample "legacy" messages to test my Outcome changes to use as backwards/forward compatibility test cases to cover this type of thing.